.github/workflows/Security Vulnerability Check on Release Tags.yml

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Security Vulnerability Check on Release Tags
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  
+  pull_request:
+    branches: [main] 
```

Comment:
Can we remove this PR trigger?

Consider adding a `workflow_dispatch`, it could come handy if we need to test the workflow, perhaps even include the tags as a parameter, although that could be too much work?

Reply:
@axelmarquezh  Please see this note https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/951#issuecomment-1944175159 . I already mentioned here .  no need  of workflow_dispatch since it's nightly job requirement in this ticket